content/how-to/rackspace-fabric/rackspace-fabric-faq/index.md

```diff
@@ -0,0 +1,64 @@
+{{</accordion>}}
+
+{{<accordion title="I am not able to login to MyRack portal. I have been asked for MFA verification though I have not configured MFA yet." col="in" href="accordion3">}}
+MFA is mandatory for all customers but should allow customer to configure it. In case to remove the MFA please contact identity team. To configure MFA, refer to [Multi-Factor Authentication](/support/how-to/multi-factor authentication)
+{{</accordion>}}
+
+{{<accordion title="I am not getting email notification via My Rackportal for incidents pertaining to Alert Logic." col="in" href="accordion4">}}
+You will receive the notification if the device in question is enabled in the **Notification Preference** option. To check the preference option, refer to [Notification-Preference](/support/how-to/notification prefernce).
+ {{</accordion>}}
+
+{{<accordion title="I am unable to login to MyRack Portal. While login it is showing no primary contact exists." col="in" href="accordion5">}}
+Create a new ticket in the fabric ticketing app under the category **Account** and select **User Management** as the account type. Contact your Admin or follow up with the support team or Account Manager if you are unable to log into the portal.
+{{</accordion>}}
+
+### Permission
+
+{{<accordion title="I am unable to upload documents and tried multiple times on multiple tickets." col="in" href="accordion1">}}
+Opt in to the ticketing beta using the ticketing drop down in the global navigation.
+{{</accordion>}}
+
+{{<accordion title="I have raised ticket using Rackspace BETA version of ticketing system, but unable to edit the ticket. I am getting the notification as I don't have permission to update." col="in" href="accordion2">}}
+Customer should raise the ticket in one of those 5 categories (Change/Incident/Account/Billing and payments/Request for information) and select appropriate account and product. 
+Also need to contact admin (customer side) or need to contact their Customer Success Manager or Support Team.
+{{</accordion>}}
+
+{{<accordion title="I need to access the Managed Backup option in the portal." col="in" href="accordion3">}}
+To access **View Reports**, you can see documentation linked below. To have access to the Managed Backup feature. https://one.rackspace.com/display/myrackspace/Managed+Backup?searchId=OLV66F2LT
+{{</accordion>}}
+
+{{<accordion title="I am having issue in loading MAR report in the portal." col="in" href="accordion4">}}
+MAR is generated at Global Data Operations side monthly which is then reported to MyRack. 
+Below is a documentation with more details:
+[MAR Process](https://one.rackspace.com/display/SMBHAMS/MAR+Processes)
+{{</accordion>}}
+
+{{<accordion title="How can I update device name in Vcenter by removing Devices IDs?" col="in" href="accordion5">}}
+The naming convention for vm should be <CoreDeviceId>-<device name>, for these vms seems this pattern is not followed so we observed that deviceid is null in our database.
```

Comment:
The convention example is not rendering on page. See if we can fix.

Reply:
Deleted <CoreDeviceId> as per https://core.rackspace.com/py/ticket/view.pt?ref_no=211209-00950.